src/settings/mod.rs

```diff
@@ -67,8 +69,6 @@ deserialize_and_validate! {
     (AwsSecret, aws_secret, "AWS secret key"),
```

Comment:
You mentioned some follow up work here to expose these as enums. Should we make a bug to track that?

Reply:
Way ahead of you: #216 :smile: